src/Umbraco.Web.UI.Client/src/packages/core/entity/types.ts

```diff
@@ -9,5 +9,17 @@ export interface UmbNamedEntityModel extends UmbEntityModel {
 	name: string;
 }
 
+// TODO: Should this be in its own package?
+export interface UmbEntityWithFlags extends UmbEntityModel {
+	flags: Array<UmbEntityFlag>;
+}
+export interface UmbEntityPotentialWithFlags extends UmbEntityModel {
```

Comment:
What do you think about the name `UmbEntityWithOptionalFlags` ?

Reply:
Yeah, good to me. I think i used that name because we in TypeScript call it Partial. But your suggestion sounds better

Reply:
oh i see that was not even the name used by me.. sorry... so optional is good for sure

Reply:
Cool. I have updated this